### ISSUE TITLE
Improve code coverage in unit tests

### ISSUE THREAD

**Issue by zimmi88** [linkedin/hopscotch#154]
Latest jasmine code coverage reports highlight some areas in which we could improve our code coverage...
- HopscotchBubble placement.
- Positioning on targeted fixed elements.
- Clicking buttons on bubbles.
- Next step selection.
- `step.delay`
- Other error cases here and there.

**Comment by kate2753** [comment]
I've added extensive set of unit test for callout positioning. These unit tests cover following config options: `placement`, `xOffset`, `yOffset`, `arrowOffset` and `isRtl`. They also include unit tests for target elements that have fixed position. See https://github.com/linkedin/hopscotch/pull/218 for more details.

This issue is pretty broad, and covers too many things that need to be tackled. It makes it hard to track the progress within the milestone. Let's consider this issue fixed when we've added unit tests for things that are explicitly called out here. New issue should be open for other areas that need test coverage.

As a side note, I've create a new issue for unit test coverage of `target` and `getTarget` options https://github.com/linkedin/hopscotch/issues/219
